src/components/reusable/pagetitle/pageTitle.ts

```diff
@@ -0,0 +1,90 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+import PageTitleScss from './pageTitle.scss';
+
+import '@kyndryl-design-system/shidoka-foundation/components/icon';
+
+/**
+ * Page Title
+ */
+
+@customElement('kyn-page-title')
+export class PageTitle extends LitElement {
+  static override styles = PageTitleScss;
+
+  /** Whether to show headline. */
+  @property({ type: Boolean })
+  showheadLine = false;
```

Comment:
These boolean "show" props might be unnecessary. Just check if the string prop is empty or not instead.

Reply:
What if they give space ?

Reply:
Done